src/Clipper.jl

```diff
@@ -534,6 +542,10 @@ function Base.length(c::__ClipperPolyTree)
     @cxx c->Total()
 end
 
+function child_count(c::__ClipperPolyTree)
+    @cxx c->ChildCount()
+end
+
```

Comment:
`PolyTree` is a subclass of `PolyNode`, so I don't think `getindex` makes sense.  See definition [here](https://github.com/Voxel8/Clipper.jl/blob/6a3cce9ea409d7b4977024e3bbee1830955cb205/src/clipper_cpp.jl#L163)